examples/solidjs-with-layout/README.md

```diff
@@ -0,0 +1,19 @@
+## solidjs-with-layout
+
+This example shows how to work work with layout (or layouts). Split layout to dumb component (with markup) and smart component for widget compositions. Dumb layout can be placed in `@share/ui/Layout`, smart - `@app/layouts/BaseLayout.tsx`. See [@app/layouts/baseLayout.tsx]() and [@shared/ui/Layout/*]() in code.
```

Comment:
```suggestion
This example shows how to work work with layout (or layouts). Split layout to dumb component (with markup) and smart component for widget compositions. Dumb layout can be placed in `@shared/ui/Layout`, smart - `@app/layouts/BaseLayout.tsx`. See `@app/layouts/baseLayout.tsx` and `@shared/ui/Layout/*` in code.
```

Reply:
Fixed (add backticks and correct link to files) 27d7518165bc794051ef17738f2f31da00e5748d